resources/multisite_drupal_standard.make

```diff
@@ -365,7 +367,7 @@ projects[i18nviews][subdir] = "contrib"
 projects[i18nviews][version] = "3.0-alpha1"
 
 projects[inline_entity_form][subdir] = "contrib"
-projects[inline_entity_form][version] = "1.6"
+projects[inline_entity_form][version] = "1.8"
```

Comment:
This module upgrade needs to pass the QA of the maintenance.
(usual workflow when we want to upgrade a module on 'master')

Reply:
ok

Reply:
Is this upgrade mandatory? If it is, what is the linked issue? Thanks!

Reply:
https://webgate.ec.europa.eu/CITnet/jira/browse/NEPCB-11
But there is not detail of what we do that...